.github/ISSUE_TEMPLATE/improvements.md

```diff
@@ -12,10 +12,11 @@ Describe more about the change.
 
 ### Domain of Docs
 
-- [ ] Solidity
-- [ ] Python
-- [ ] JavaScript
-- [ ] Readme
+- [] Readme
```

Comment:
Remove 16 , 17 ,and 18 and add this ->
 [ ] Solidity
[ ] Python
[ ] JavaScript
[ ] Golang

Reply:
@surjyaneeh  update all this